src/lib.rs

```diff
@@ -638,6 +657,77 @@ pin_project_lite::pin_project! {
     /// If a notified listener is dropped without receiving a notification, dropping will notify
     /// another active listener. Whether one *additional* listener will be notified depends on what
     /// kind of notification was delivered.
+    ///
+    /// The listener is not registered into the linked list inside of the [`Event`] by default. It
```

Comment:
Nit: we should make it explicit here that we're talking about EventListener created using `new` method and not `Event::listen`. "by default" is a bit unclear.